README.md

```diff
@@ -29,14 +29,13 @@ At this point, the development environment should be completely setup and the Ka
     rails s
     ```
 
-2. Access Foreman in your browser (e.g. `http://<hostname>:3000/`)
-3. Login to Foreman (default: `admin` and `changeme`)
-4. Create an initial Foreman organization
-5. Navigate to the Katello engine (e.g. `http://<hostname>:3000/katello`)
+1. Access Foreman in your browser (e.g. `http://<hostname>:3000/`)
+1. Login to Foreman (default: `admin` and `changeme`)
+1. If you go to `http://<hostname>:3000/about` and view the "Plugins" tab, you should see a "Katello" plugin listed.
```

Comment:
Would you mind changing these URL's to `https://<hostname>`? The katello-devel installer actually sets up an apache reverse proxy to :3000.